### ISSUE TITLE
Update to build correctly with Rack 2.4.0 SDK

### ISSUE THREAD

**Issue by pgatt** [dustinlacewell/vcv-prefabs#7]
Hi Dustin,

Tried building this and ran into issues. Would you mind building for the new SDK?

**Comment by baconpaul** [comment]
https://github.com/baconpaul/airwin2rack/commit/bc39556738e0bf27af1390d49aa77f2e0b886041 May help

**Comment by pgatt** [comment]
Yep that did it, though there were other errors, I think I managed to fix them here, but it may still be worth taking a quick look.
